generators/chipyard/src/main/scala/SpikeTile.scala

```diff
@@ -16,7 +16,9 @@ import freechips.rocketchip.util._
 import freechips.rocketchip.tile._
 import freechips.rocketchip.prci._
 
-case class SpikeCoreParams() extends CoreParams {
+case class SpikeCoreParams(
+  nPMPs: Int = 16
```

Comment:
I'd prefer to just make this a hard default to 0 in the params, rather than introducing a default that is seldom useful

Reply:
If we are trying to match Rocket w/ the configurations maybe this should be set to 8 by default (what Rocket does). IMO it's better to match Rocket then have a specific checkpointing config that has the right arch. state set than have some implicitly set default value that works for one config (Spike) but not for the other (Rocket) when reloading checkpoints.